javascript_nodejs/18.multi-lingual/README.MD

```diff
@@ -0,0 +1,60 @@
+# Multilingual Bot Sample
+
+## Concepts introduced in this sample
+
+Translation Middleware: We create a translation middleware than can translate text from bot to user and from user to bot, allowing the creation of multi-lingual bots. 
```

Comment:
remove `'-'`, and the folder and .bot files also need to be renamed to `multilingual`